test/utils/setupKarma.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-env mocha */
```

Comment:
This file could be merged into `test/karma.test.js`, no? It only contains a dozen of lines. The core version is much bigger and makes sense the split.

Reply:
It was to follow the core file structure but yes our version could have a single file

Reply:
Done